<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>qSlicerEMSegmentGraphWidget</class>
 <widget class="qSlicerEMSegmentWidget" name="qSlicerEMSegmentGraphWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>528</width>
    <height>290</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <widget class="QComboBox" name="Input0ComboBox"/>
   </item>
   <item row="0" column="1">
    <layout class="QHBoxLayout" name="Input0Layout"/>
   </item>
   <item row="1" column="0" colspan="2">
    <widget class="QVTKWidget" name="Graph0Widget" native="true">
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>100</height>
      </size>
     </property>
    </widget>
   </item>
   <item row="2" column="0">
    <widget class="QComboBox" name="Input1ComboBox"/>
   </item>
   <item row="2" column="1">
    <layout class="QHBoxLayout" name="Input1Layout"/>
   </item>
   <item row="3" column="0" colspan="2">
    <widget class="QVTKWidget" name="Graph1Widget" native="true">
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>100</height>
      </size>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>qSlicerWidget</class>
   <extends>QWidget</extends>
   <header>qSlicerWidget.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>qSlicerEMSegmentWidget</class>
   <extends>qSlicerWidget</extends>
   <header>qSlicerEMSegmentWidget.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>QVTKWidget</class>
   <extends>QWidget</extends>
   <header>QVTKWidget.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
